SPECS/virtiofsd/config.toml

```diff
@@ -0,0 +1,5 @@
+[source.crates-io]
```

Comment:
Is this a standard way of adding a config for vendored sources? Comparing with the way we vendor kata-cc, the directory .config/ and file config.toml are included in the "cargo.tar.gz". So for those cases there isn't a need to include this config

Reply:
Discussed offline but this is following cloud-hypervisor: https://github.com/microsoft/CBL-Mariner/blob/3798a92d85758462fd12f7fa8abce93802de6234/SPECS/cloud-hypervisor/cloud-hypervisor.spec#L16-L23